optimade/server/mappers/entries.py

```diff
@@ -1,10 +1,11 @@
+import abc
 from typing import Tuple
 from optimade.server.config import CONFIG
 
 __all__ = ("ResourceMapper",)
 
 
-class ResourceMapper:
+class ResourceMapper(abc.ABC):
```

Comment:
I don't think this class needs to be a meta-class?
```suggestion
class ResourceMapper:
```

Reply:
I guess not, it was more to signal that we expect it to be overridden, not used directly by an implementation. I'll leave it for now and we can undo this before merging, if desired.

Reply:
Sure. We could rename it to `BaseResourceMapper` or `GenericResourceMapper` and write in the doc-string that it's meant to be sub-classed.

Another issue here though, and more general, we are getting more and more Mongo-specific code into the "general" code. Especially in this PR.

Reply:
> Sure. We could rename it to `BaseResourceMapper` or `GenericResourceMapper` and write in the doc-string that it's meant to be sub-classed.
> 
> Another issue here though, and more general, we are getting more and more Mongo-specific code into the "general" code. Especially in this PR.

Yeah, agreed, I was a little worried about this too... the other option is to check the aliases on init of the `MongoCollection` which might be more sensible.